static/css/com/layout/header.scss

```diff
@@ -94,29 +104,31 @@
 }
 
 .global-header-panel {
-  @include clearfix;
   box-sizing: border-box;
-  padding: $unit 0;
-  height: 60px;
+  display: flex;
+  width: 100%;
+  padding: 0 22px;
+  align-items: center;
+  justify-content: space-between;
+  height: 64px;
```

Comment:
Same question?

Reply:
We need it to set header height.
And we can't use min-height cause we have banner in this header :(